PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,50 @@
+## Description
+
+*What does this PR do?*
```

Comment:
A brief 1- or 2-sentence high-level summary is appropriate here, and optionally can expand on that.